services/dy-mapcore-widget/Dockerfile

```diff
@@ -0,0 +1,108 @@
+FROM node:8.15.0-alpine as base
+
+LABEL maintainer=odeimaiz
+
+# non-root user 'scu'
+RUN adduser -D -u 8004 scu
+
+RUN apk add --no-cache \
+    su-exec
+
+ENV HOME /home/scu
+
+EXPOSE 4000
+
+WORKDIR $HOME
+
+RUN apk update &&\
+    apk add --no-cache \
+    python3 \
+    bash
+
+
+RUN pip3 install --upgrade pip wheel setuptools
+
+# ----------------------------------------------------------------
+# install simcore-sdk
+RUN apk add --no-cache --virtual .build-deps \
+    g++ \
+    git \
+    python3-dev \
+    postgresql-dev && \
+    pip install --no-cache-dir git+https://github.com/ITISFoundation/osparc-simcore.git@master#subdirectory=packages/service-library &&\
```

Comment:
please freeze these as well as the dependencies . Actually,  Do we really need them?

Reply:
This is the result of a copy-pasting, I don't even know if we need them.

It works for me, if you want to clean it up, feel free to take it over.

Reply:
:-(

Reply:
https://github.com/ITISFoundation/osparc-issues/issues/256